lib/reek/configuration/configuration_converter.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require_relative './configuration_validator'
+
+module Reek
+  module Configuration
+    # Responsible for converting marked strings coming from the outside world
+    # into proper regexes.
+    class ConfigurationConverter
```

Comment:
This module seems used but not tested ...

Reply:
It's tested via configuration_file_finder_spec because that is where the tests resided for the existing "string to regex" feature. I'm happy to refactor this if you think that this rather belongs into configuration_converter_spec but I'd appreciate it if we could do this via separate issue / PR to not make this PR more messy than it already is.